File: test/fixtures/output/go/native/full.go
package main

import (
	"fmt"
	"strings"
	"net/http"
)

func main() {
	client := &http.Client{}
	url := "http://mockbin.com/har?baz=abc&foo=bar&foo=baz"
	body := "foo=bar"
	req, _ := http.NewRequest("POST", url, strings.NewReader(body))
	req.Header.Add("Accept", "application/json")
	req.Header.Add("Content-Type", "application/x-www-form-urlencoded")
	req.Header.Add("Cookie", "foo=bar; bar=baz")
	res, _ := client.Do(req)
	fmt.Printf("%+v", res)
}

File: test/fixtures/output/ruby/native/full.rb
require 'uri'
require 'net/http'

url = URI("http://mockbin.com/har?foo=bar&foo=baz&baz=abc&key=value")

conn = Net::HTTP.new(url.host, url.port)

request = Net::HTTP::Post.new(url)
request["cookie"] = 'foo=bar; bar=baz'
request["accept"] = 'application/json'
request["content-type"] = 'application/x-www-form-urlencoded'
request.body = "foo=bar"

response = conn.request(request)
puts response.read_body
